<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateYACS</class>
 <widget class="QDialog" name="CreateYACS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create YACS</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="Name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="LEName">
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Case">
       <property name="text">
        <string>Case</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="PBCaseName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="LECaseName">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Script">
       <property name="text">
        <string>Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBScriptFile">
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEScriptFile">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="DirectoryStart">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBDir">
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEDirName">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="MeshFile">
       <property name="text">
        <string>Mesh file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBMeshFile">
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEMeshFile">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="GBTypeSchema">
     <property name="title">
      <string>Type of schema</string>
     </property>
     <layout class="QHBoxLayout" name="_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RBConstant">
        <property name="text">
         <string>Constant</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBVariable">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="GroupButtons">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonOk">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>128</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
